### ISSUE TITLE
Dates and Tasks are not shown in calendar

### ISSUE THREAD

**Issue by susnux** [gggard/AndroidCaldavSyncAdapater#133]
I want to sync my webdav calender (using Horde 5.1.4), when I create the account, the app get the calender (Found new Calendar) and in the Google Calendar App at shown calendars the Calendar is activated.
But the dates are not shown. So I do not see them in the calendar, but when I create a new date on the phone it get synchronized with the server so this way works...

Using Android 4.3 (Nexus 4) and App version 1.8
Server: Horde 5.1.4

**Comment by timoberger** [comment]
please try to use another calendar app like aCalendar: https://play.google.com/store/apps/details?id=org.withouthat.acalendar
are the events visible there?

**Comment by susnux** [comment]
No, they are also not visible in aCalendar...

**Comment by timoberger** [comment]
can you provide a test account for horde?

**Comment by susnux** [comment]
Sorry, but I use our university server, so I can not provide one.
But it is working perfect with Thunderbird (Lightning).

**Comment by timoberger** [comment]
- which tool created the events that you can't see?
- when events are created by thunderbird, are they synced to your device?
- when events are created by your device, are they synced to thunderbird?

**Comment by susnux** [comment]
I created events with Thunderbird and with the web-frontend of horde.
So Thunderbird works perfect, also when I creat events with my phone, the events get synced with the server.
So "upload" works, but "download" does not work...

**Comment by timoberger** [comment]
so you are able to see only events on your device that you created with the device?
what happens when you delete and recreate the account on your device? are the events visible again or do they disappear like any other event?

**Comment by susnux** [comment]
Ok, I created an event.
1. when I delete the account it disappear (I think it should do it ;) ).
BUT: Now (after deleting the account all events from the webdav appear.... I found a reason therefor: I also use the E-Mail function of Horde by using an exchange account, so after deleting webdav adapter the exchange account synced the events...

So I deleted also the exchange account and recreate the webdav account:
The same like before: upload works, download does not.

**Comment by timoberger** [comment]
i will test the server demo.horde.org (when he is reachable again)

**Comment by timoberger** [comment]
please check whether you can access this calendars
server: http://demo.horde.org/rpc.php/principals/demo/
user: demo
pass: demo

**Comment by susnux** [comment]
The calendars are found, but it synchronizes until now... and no events are shown...

**Comment by timoberger** [comment]
which version and source of the sync adapter are you using?
device? android version? cyanogenmod?

please try to add the demo account by owncloud
server: http://demo.owncloud.org/remote.php/caldav/principals/test
user: test
pass: test

**Comment by sh2012** [comment]
I have the same problem. Yesterday installed a new eclipse package with adt just to test the syncadapter. I am using a new owncloud server (5.x) and thunderbird/lightning as another client. Good: Synchronizing events created in the android client to the owncloud server works. Bad: All events directly created in the owncloud webinterface or with the thunderbird client are not synchronized to the android client.

**Comment by timoberger** [comment]
@sh2012 which version and source of the sync adapter are you using? device? android version? cyanogenmod?

**Comment by susnux** [comment]
PS: I use the latest version from google Play and a Nexus 4 with Android 4.3 (normal from google).

**Comment by sh2012** [comment]
I am using last version from gihtub (1.8), and the eclipse (4.3.1) emulator (galaxy nexus, 720x1280) with android version 4.2.2 (api level 17).

**Comment by timoberger** [comment]
did you install any other sync adapter on your devices?

**Comment by sh2012** [comment]
No. Just a fresh install (Eclipse downloaded from Eclipse.org and Android SDK from the Google Sites)

**Comment by timoberger** [comment]
both of you are using the nexus 4. i guess this is a device depending problem.

**Comment by timoberger** [comment]
wiki: https://github.com/gggard/AndroidCaldavSyncAdapater/wiki/device-specific-problems

**Comment by timoberger** [comment]
i checked the current version against the avd "Nexus 4" with API 18 without problems.
